charts/soroban-rpc/templates/soroban-rpc-cm.yaml

```diff
@@ -0,0 +1,212 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "common.fullname" . }}
+  {{- if .Release.Namespace }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels:
+    app: {{ template "common.fullname" . }}
+    chart: {{ template "common.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  soroban-rpc.cfg: |
+    ENDPOINT={{ .Values.sorobanRpc.sorobanRpcConfig.endpoint | default "0.0.0.0:8000" | quote }}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).friendbotUrl -}}
+    FRIENDBOT_URL={{ .Values.sorobanRpc.sorobanRpcConfig.friendbotUrl | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).networkPassphrase -}}
+    NETWORK_PASSPHRASE={{ .Values.sorobanRpc.sorobanRpcConfig.networkPassphrase | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).stellarCoreUrl -}}
+    STELLAR_CORE_URL={{ .Values.sorobanRpc.sorobanRpcConfig.stellarCoreUrl | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).captiveCoreConfigPath -}}
```

Comment:
Yeah, very good points. I'll remove those.

What's your opinion on the following:
`stellarCoreUrl` - we could hardcode to `http://localhost:11626` since we run local core
`dbPath` - hardcode to `/var/lib/stellar/soroban-rpc-db.sqlite`
`stellarCaptiveCorePort` - hardcode to 11626

and in core config:
`peerPort` - hardcode to 11625
`database` - hardcode to `sqlite3:///var/lib/stellar/stellar-captive-core.db`
`bucketlistDb` - hardcode to true
`httpPort` - hardcode to 11626

I'm not sure about `publicHttpPort`, maybe it makese sense to keep it?

Reply:
yes, those additional default recommendations sound good, remove as chart params. 

`stellarCoreUrl` - I think can skip as param, and don't set it internally, I think rpc ingest will then default it to localhost url that it derives from captive core's `httpPort` in .cfg

I think can hardcode` publicHttpPort=true`, since it then triggers `httpPort=11626` to be created by the captive core process. b/c I think rpc is submitting tx's to `stellarCoreUrl`

Reply:
Made the updates as per above! Thanks again for good suggestions.